pkg/netpol/internal/dotformatting/dot_output_formatting.go

```diff
@@ -0,0 +1,54 @@
+package dotformatting
+
+import (
+	"sort"
+	"strings"
+)
+
+// common dot output formatting consts and funcs
+const (
+	DotHeader  = "digraph {"
+	DotClosing = "}"
+)
+
+// AddPeerToNsGroup adds the peer line to the namespace list in the given map.
+func AddPeerToNsGroup(peerNs, peerLine string, mapNsToPeers map[string][]string) {
+	if _, ok := mapNsToPeers[peerNs]; !ok {
+		mapNsToPeers[peerNs] = []string{}
+	}
+	mapNsToPeers[peerNs] = append(mapNsToPeers[peerNs], "\t"+peerLine) // adding tab to the subgraph's lines
+}
+
+// return the label of a cluster's peer to be displayed on its node in the dot graph
+func NodeClusterPeerLabel(name, kind string) string {
+	return name + "[" + kind + "]"
+}
+
+// AddNsGroups adds namespaces frames to dot graphs
```

Comment:
how does it add? what is the input/output, and what it does with it?

Reply:
updated the comment